yswift/Sources/Transactable.swift

```diff
@@ -0,0 +1,18 @@
+import Yniffi
+
+protocol Transactable {
+    var document: YDocument { get }
+    func inTransaction<T>(_ transaction: YrsTransaction?, changes: @escaping (YrsTransaction) -> T) -> T
```

Comment:
Naming nit: It's a bit more common to see the pattern `withTransaction` in idiomatic Swift API.

Reply:
Totally agree on this one! It was the name I initially started with, but for some-already-obscure-reason I decided to rename it to `inTransaction` 

Pushed the changes renaming from `inTransaction` back to `withTransaction`